frontend/src/axios.tsx

```diff
@@ -439,6 +440,22 @@ export class APIClient {
       });
   }
 
+  public LibrarianSignup(data: SignupData): Promise<Tokens> {
+    return APIClient.axiosInstance.post("/auth/librarian/register", data)
+      .then((res) => {
+        let tokens: Tokens = {
+          access_token: res.data.data.access_token,
+          refresh_token: res.data.data.refresh_token,
+        };
+        localStorage.setItem("access_token", tokens.access_token);
+        localStorage.setItem("refresh_token", tokens.refresh_token);
+        APIClient.instance.SetAuthorizationHeaders(
```

Comment:
We do not need to change the authentication header here. This basically means that we will be using the token of the newly created librarian now onwards, but that is not the case

Reply:
Humm, Similarly I also don't think that we need to replace token in local storage for the same reason @mustansirMuz  mentioned.

Reply:
DONE